optaplanner-examples/src/main/java/org/optaplanner/examples/coachshuttlegathering/domain/Coach.java

```diff
@@ -42,6 +47,13 @@ public void setDestination(BusHub destination) {
     // Complex methods
     // ************************************************************************
 
+    @CustomShadowVariable(variableListenerClass = CoachPassengerQuantityTotalUpdatingVariableListener.class,
+            sources = { @PlanningVariableReference(entityClass = BusStop.class, variableName = "bus") })
+    @Override
+    public Integer getPassengerQuantityTotal() {
+        return super.getPassengerQuantityTotal();
+    }
```

Comment:
This is sort of unnatural, and definitely unexpected. I had to override the method, because the annotation would not be inherited.